amethyst_test/src/amethyst_application.rs

```diff
@@ -131,14 +131,19 @@ where
     /// Returns the built Application.
     ///
     /// If you are intending to call `.run()` on the `Application` in a test, be aware that on
-    /// Linux, this will cause a segfault when `RenderBundle` is added and GL is using software
-    /// rendering, such as when using Xvfb or when the following environmental variable is set:
+    /// Linux, this will cause a segfault when `RenderingSystem` is added and GL software rendering
```

Comment:
there is no GL in the rendering pipeline :P

Reply:
ah, i know, but was lazy to remove the comment for the future-dev when `glow` is integrated (and sift through git history) 😛

Reply:
fixed!